apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregningsgrunnlag/InstitusjonsoppholdBP.tsx

```diff
@@ -60,7 +60,13 @@ const InstitusjonsoppholdBP = (props: InstitusjonsoppholdProps) => {
   const feilOverlappendePerioder: [number, FeilIPeriode][] = [
     ...feilIKomplettePerioderOverIntervallInstitusjonsopphold(heleSkjemaet),
   ]
-  const ferdigstilleForm = (data: { institusjonsOppholdForm: InstitusjonsoppholdGrunnlagData }) => {
+  useEffect(() => {
```

Comment:
Vil nå være sånn at å fjerne alle perioder lagrer automatisk, mens alle andre endringer må lagres manuelt?

Reply:
Ingenting lagres før man trykker "neste side" uansett årsak, dette er bare for å få med riktig data når man gjør det.